src/contracts/pods/EigenPod.sol

```diff
@@ -428,6 +451,96 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
         _sendETH(recipient, amountWei);
     }
 
+    function requestPartialWithdrawalsProof(
+        uint64 oracleTimestamp,
+        uint64 endTimestamp,
+        address recipient,
+        uint32 callbackGasLimit
+    ) external onlyEigenPodOwner hasEnabledRestaking {
+        requestNonce++;
+        require(endTimestamp > timestampProvenUntil, "EigenPod.submitPartialWithdrawalsBatchForVerification: endTimestamp must be greater than timestampProvenUntil");
+        
+        //the proofs are made [startTimestamp, endTimestamp), so the next startTimestamp = previous endTimestamp     
+        uint64 startTimestamp = timestampProvenUntil;
+        
+        //record the partial withdrawal proof request
+       _partialWithdrawalProofRequests[requestNonce] = PartialWithdrawalProofRequest({
+            startTimestamp: startTimestamp,
+            endTimestamp: endTimestamp,
+            recipient: recipient,
+            status: REQUEST_STATUS.PENDING
+       });
+
+
+
+        emit PartialWithdrawalProofRequested(startTimestamp, endTimestamp, requestNonce);
+
+        //NOTE: it is not clear what callback data needs to passed here since succinct is going to make multiple callbacks for each request.
+        //These are the inputs to the callback function for this specific proof request.
```

Comment:
is the splitting based upon amount of proofs?
seems like a notable design problem then, at minimum.
would be easy if the splitting was just based on a time/slot length